### PR TITLE
Move to CMake targets

### PR DESCRIPTION
**Pull request by awegrzyn**

### CODE REVIEW

File: cmake/InfoLoggerConfig.cmake.in [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@PACKAGE_INIT@
+
+get_filename_component(InfoLogger_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+include(CMakeFindDependencyMacro)
+
+list(APPEND CMAKE_MODULE_PATH ${InfoLogger_CMAKE_DIR})
+
+find_dependency(Common HINTS /Users/awegrzyn/Common_install)
```
Comment (aphecetche) [inline cmake/InfoLoggerConfig.cmake.in]: I guess this is wrong, isn't it ?

BTW, I don't quite get why you need the get_filename_component  + list(append + list(remove_at) bits ? Looks like a "trick"

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
 find_package(Common REQUIRED)
 find_package(MySQL)
 find_package(PythonLibs)
+get_property(Common_PATH TARGET AliceO2::Common PROPERTY LOCATION)
+message(${CMAKE_MODULE_PATH})
+
+set(LIBRARY_OUTPUT_PATH "${CMAKE_BINARY_DIR}/lib")
+set(EXECUTABLE_OUTPUT_PATH "${CMAKE_BINARY_DIR}/bin")
+set(INCLUDE_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/include")
+set(CMAKE_INSTALL_LIBDIR lib)
```
Comment (aphecetche) [inline CMakeLists.txt]: This last one is to insure there's no lib64 on some platform, is that right ?

The other 3 I don't understand : aren't they the default ? Why re-specifying them then ?

File: cmake/InfoLoggerConfig.cmake.in (hunk 1 above)
Comment (awegrzyn) [reply cmake/InfoLoggerConfig.cmake.in]: Hi, Indeed it's not needed anymore

File: CMakeLists.txt (hunk 2 above)
Comment (awegrzyn) [reply CMakeLists.txt]: They are not really default, e.g. If `EXECUTABLE_OUTPUT_PATH` is not set, test executables will not be moved to `bin` directory.

**Comment by awegrzyn** [comment]
https://github.com/AliceO2Group/InfoLogger/pull/11

**Pull request closed without merging**